BusinessLogic/Entities/RoleRegistry.cs

```diff
@@ -12,5 +13,18 @@ public static class RoleRegistry
         public static readonly IdentityRole Admin = new IdentityRole { Id = "1", Name = "Admin", NormalizedName = "ADMIN" };
         public static readonly IdentityRole Doctor = new IdentityRole { Id = "2", Name = "Doctor", NormalizedName = "DOCTOR" };
         public static readonly IdentityRole Patient = new IdentityRole { Id = "3", Name = "Patient", NormalizedName = "PATIENT" };
+
+        public static IReadOnlyList<IdentityRole> GetRoles()
+        {
+            Type type = typeof(RoleRegistry);
+            FieldInfo[] fields = type.GetFields(BindingFlags.Public | BindingFlags.Static);
+            var roles = new List<IdentityRole>();
+            foreach (FieldInfo field in fields)
+            {
+                var role = (IdentityRole)field.GetValue(null);
+                roles.Add(role);
+            }
+            return roles;
+        }
```

Comment:
Use private field to store them, bc it will be the same after startup, but u dynamically load it. U can load it once on startup and memoize